### PR TITLE
Completed card hover project

### PR DESCRIPTION
**Pull request by Naufil-Shemle**

### CODE REVIEW

File: tasks-by-users/Naufil-Shemle/05-card-hover-effect/style.css [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  background: rgb(211, 206, 206);
+
+  padding: 20px;
+
+  border-radius: 20px 20px 20px 20px;
```
Comment (Abbas4tech) [inline tasks-by-users/Naufil-Shemle/05-card-hover-effect/style.css:97]:
```suggestion
  border-radius: 20px;
```

File: tasks-by-users/Naufil-Shemle/05-card-hover-effect/style.css [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+  position: relative;
+
+  display: flex;
+
+  justify-content: center;
+  align-items: center;
+
+  background: #333;
+
+  border-radius: 20px 20px 20px 20px;
```
Comment (Abbas4tech) [inline tasks-by-users/Naufil-Shemle/05-card-hover-effect/style.css:50]:
```suggestion
  border-radius: 20px;
```

File: tasks-by-users/Naufil-Shemle/05-card-hover-effect/style.css [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+  background: #333;
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+img {
+  width: 128px;
+  filter: invert(100%);
```
Comment (Abbas4tech) [inline tasks-by-users/Naufil-Shemle/05-card-hover-effect/style.css:16]: What does this Invert property do? @Naufil-Shemle

File: tasks-by-users/Naufil-Shemle/05-card-hover-effect/style.css [hunk 4]
```diff
@@ -0,0 +1,135 @@
+body {
+  margin: 0;
+  padding: 0;
```
Comment (Abbas4tech) [inline tasks-by-users/Naufil-Shemle/05-card-hover-effect/style.css:3]:
```suggestion
* {
  margin: 0;
  padding: 0;
  Box-sizing: border-box;
  }
 body {
```

**Review by Abbas4tech — CHANGES_REQUESTED** [review]

**Review by Tanvir04khan — APPROVED** [review]

File: tasks-by-users/Naufil-Shemle/05-card-hover-effect/style.css (hunk 3 above)
Comment (Naufil-Shemle) [reply tasks-by-users/Naufil-Shemle/05-card-hover-effect/style.css:19]: Icon was in black color so I wanna changes its color to white that's why I used this property

**Review by shaikh-7abish — APPROVED** [review]

**Review by Tauqeer-Ahmed-99 — APPROVED** [review]

**Review by syedabdullahali — COMMENTED** [review]
`nice`

**Review by Abbas4tech — APPROVED** [review]

**Pull request merged**